Helper/Data.php

```diff
@@ -101,25 +104,31 @@ protected function _getConfigValue($path, $contextScope, $contextScopeId) {
      * @param string|int $contextScopeId
      * @return array
      */
-    public function getConfigDisplayValue($path, $contextScope, $contextScopeId) {
+    public function getConfigDisplayValue($path, $contextScope, $contextScopeId)
+    {
         $value = $this->_getConfigValue($path, $contextScope, $contextScopeId);
 
-        $labels = [$value]; //default labels to raw value
+        if (is_array($value)) {
+            $firstKey = array_keys($value)[0];
+            $labels = [$value[$firstKey]];
```

Comment:
You could also retrieve the first value by using [current](https://php.net/current) if it's a "fresh" array.
```suggestion
            $labels = [current($value)];
```

Reply:
Actually, I would not recommend to use `current()`. If you modify code above this function which causes the array pointer to be somewhere else then at the start of the array (for example: a `foreach()`), `current()` might suddenly return something else then the first value of this array.

Reply:
Since PHP 7 [foreach](https://php.net/foreach) does not use the internal array pointer anymore.

You might otherwise consider using [reset()](https://php.net/reset) which resets the array and returns the first entry.